src/docs/spec-model.md

```diff
@@ -81,3 +81,66 @@ In addition, predicates from the following sources MAY also be encouraged:
 
 * any relation from the [Relation Ontology (RO)](https://obofoundry.org/ontology/ro.html);
 * any relation under [skos:mappingRelation](http://www.w3.org/2004/02/skos/core#mappingRelation) in the [Semantic Mapping Vocabulary](https://mapping-commons.github.io/semantic-mapping-vocabulary/).
+
+## Non-standard slots
+
+<a id="non-standard-slots"></a>
+
+Implementations are only REQUIRED to support the standard metadata slots defined in the LinkML model.
```

Comment:
> defined in the SSSOM LinkML schema

So to not cause confusion that we re talking about the sssom schema here (rather than the LinkML meta-schema)

Reply:
I wonder how could someone interpret “the LinkML model” in this context as referring to anything else than “the model defined by the LinkML schema we are talking about since the beginning of this document”, but fine, if you believe there is a possible ambiguity here, happy to clear it out.

Reply:
Thanks for accomodating, this was my impression on my first read.